### ISSUE TITLE
Unable to execute integration spec files error thrown when calling c.visit

### ISSUE THREAD

**Issue by azaeng04** [jscutlery/devkit#141]
Create an Angular application with the Cypress schematic in the documentation.
Create an integration spec with the `cy.visit('/')` command.
Launch Cypress as per usual: `yarn cypress open --project .`
Select the test to execute.

![image](https://user-images.githubusercontent.com/13921692/145388361-dc5b1d7e-4461-4fcc-92d0-73855f0ffd73.png)

CURRENT BEHAVIOUR: Exception is thrown indicating that you are unable to execute cy.visit
EXPECTED BEHAVIOUR: Exception should not throw for a usual integration spec

**Comment by edbzn** [comment]
Hi, there's no need to navigate to a URL, you don't need to call `cy.visit()` in your test. Cypress provides a canvas where the component is mounted in isolation.

**Comment by azaeng04** [comment]
Yes I understand that, but Cypress isn't allowing me to execute my standard `spec.ts` files. Shouldn't it only throw that error if I am trying to do it inside a `cy-spec.ts` file that was defined?
I am getting that error when executing it in my `spec.ts` files.
